addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -2820,6 +2820,10 @@ const SnippetOptionWidget = Widget.extend({
 
         // Ask a mutexed snippet update according to the widget value change
         const shouldRecordUndo = (!previewMode && !ev.data.isSimulatedEvent);
+        const isOptionsTab = widget.$el.closest([
+            'we-customizeblock-option.snippet-option-OptionsTab',
+            'we-customizeblock-option.snippet-option-ThemeColors',
+        ].join(',')).length;
```

Comment:
- Mmh I remember the problem was during the edition of gradients for presets, right? Maybe it is not worth a change in stable where that option does not exist. I don't think there is a problem of interacting with options while the page is being updated... it actually works and that's part of the reason why we have a mutex on editor actions, to ensure options are applied in the order they were chosen. Even in simple cases like "choose postcard layout" "oops you clicked on boxed instead" -> you can directly reopen the select and even choose the postcard layout while the boxed one was not even fully applied yet. I wonder if even in master the fix should not be at colorpicker level instead of this? :thinking: 
- If we do the fix: `widget.$el.closest(...).length` is not it the same as `this.$el.is(...)` ?
- Instead of relying on the option DOM, can't you do `this.iWantALoaderOnThePanelToo` and set that as a property on the class definition?

Reply:
Made it specific to the three color pickers in #75680, it is indeed way less intrusive.
This PR can be closed if we agree.

Reply:
:+1: